### ISSUE TITLE
FortiOS

### ISSUE THREAD

**Issue by jerebernard** [ytti/oxidized#1594]
Hello,
For a FortiWeb the backup doesn't work.

```
# # COMMAND: get system status
# International Version: FortiWeb-600D 5.83,build1388,170728
# Serial-Number: xxxx
# Bios version: 00010001
# Log hard disk: Available
# Hostname: xxxx
# Operation Mode: Reverse Proxy
# FIPS-CC mode: disabled
# Current HA mode: standalone
# 
# # COMMAND: get hardware status
# Parsing error at 'hardware'. err=1
# Command fail. CLI parsing error.
# COMMAND: show full-configuration | grep
# Parsing error at 'hardware'. err=1
# Command fail. CLI parsing error.
```
In the script forti.rb if I remove the ' | grep' everything works fine

**Comment by Ray-Zuniga-M451** [comment]
If using a FAZ the command for showing the full config is
`show`
Not related but incase it helps anyone

**Comment by jerebernard** [comment]
problem solved with a firmware update to 6.02 on the Fortiweb device
